### PR TITLE
Quotes page

### PR DESCRIPTION
**Pull request by JordanDDisch**

### CODE REVIEW

File: lib/node-detail/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ app.set('view engine', 'hbs');
 
 var cacheLong = 20;
 
-app.get('/:path', routeCache.cacheSeconds(cacheLong), function (req, res) {
+app.get('/page/:path', routeCache.cacheSeconds(cacheLong), function (req, res) {
```
Comment (rcaracaus) [inline lib/node-detail/index.js]: Still needs correct route

File: lib/hbs/index.js [hunk 2]
```diff
@@ -5,6 +5,26 @@ var hbs = exphbs.create({
     extname: ".hbs"
 });
 
+
 hbs.handlebars.registerHelper(layouts(hbs.handlebars));
+hbs.handlebars.registerHelper("truncate", function (str, limit, omission) {
```
Comment (rcaracaus) [inline lib/hbs/index.js]: preferable in separate module

File: lib/quotes/quotes.hbs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                    {{#if this.url}}
+                        <a href="{{this.url}}">
+                        ~ {{this.name}}</a>
+                    {{else}}
+                        <div>~ {{this.name}}</div>
+                    {{/if}}
+                    </div>
+                    {{#if this.product.url}}
+                    <div class="quote-listing__product__title">
+                        <a href="{{this.product.url}}">{{truncate this.product.title 25 "..."}}</a>
```
Comment (rcaracaus) [inline lib/quotes/quotes.hbs]: does this get appended if less than 25?

Comment (JordanDDisch) [inline lib/quotes/quotes.hbs]: no, if the length of the string is less then the limit set then it wont show.

File: lib/hbs/index.js [hunk 4]
```diff
@@ -1,10 +1,13 @@
 var exphbs = require("express-handlebars");
 var layouts = require("handlebars-layouts");
+var truncate = require("../helpers");
```
Comment (rcaracaus) [inline lib/hbs/index.js]: why var different to module name?

**Pull request merged**